### PR TITLE
3109: Opening hours expand options

### PR DESCRIPTION
**Pull request by b14sha**
https://platform.dandigbib.org/issues/3109

### CODE REVIEW

**Comment by cableman** [comment]
Please look into the Scrutinizer errors!

File: themes/ddbasic/scripts/ddbasic.common.js [hunk 1]
```diff
@@ -20,11 +20,21 @@
         text.push(Drupal.t('Opening hours'));
       }
 
-      $('<a />', {
-        'class' : 'opening-hours-toggle js-opening-hours-toggle js-collapsed collapsed',
-        'href' : Drupal.t('#toggle-opening-hours'),
-        'text' : text.join(', ')
-      }).insertBefore(this);
+      if (Drupal.settings.ding_ddbasic_opening_hours && Drupal.settings.ding_ddbasic_opening_hours.expand_all_libraries) {
```
Comment (cableman) [inline themes/ddbasic/scripts/ddbasic.common.js]: A better way to test if properties exits would be to use hasOwnProperty().

File: themes/ddbasic/scripts/ddbasic.common.js [hunk 2]
```diff
@@ -50,8 +60,8 @@
       event.preventDefault();
     });
 
-    // Expand opening hours on library pages.
-    if (Drupal.settings.ding_ddbasic_opening_hours && Drupal.settings.ding_ddbasic_opening_hours.expand_on_library) {
+    // Expand opening hours on first library on library pages.
+    if (Drupal.settings.ding_ddbasic_opening_hours && Drupal.settings.ding_ddbasic_opening_hours.expand_on_first_library) {
```
Comment (cableman) [inline themes/ddbasic/scripts/ddbasic.common.js]: A better way to test if properties exits would be to use hasOwnProperty().

File: themes/ddbasic/scripts/ddbasic.common.js (hunk 1 above)
Comment (cableman) [reply themes/ddbasic/scripts/ddbasic.common.js]: You could also preform the test once (as its used more than once in the code) and put it into a variable.

**Comment by JrgenGNielsen** [comment]
Can you fix the Scrutinizer error on hasOpeningHours definition?

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Pull request merged**